tasks/noauto_create_backup_user_and_group.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Setup backup environment
+  tags:
+    - install_backup
```

Comment:
No, we were talking about my crude implementation to be able to deselect the backup in complex scenarios with many roles. You showed me a nice solution with tags.
This tag is just to start the backup part again in case of multiple roles.

I do not know what speaks against it.

Ich habe noch eine Beschreibung in der Readme.md eingefügt.

Reply:
This adds a lot of clutter and it seems more practical to deselect a specific role (or call to a role) at that level. Also haven't seen this used in other roles. Can be added as separate PR later.